web/translation/translate.en_US.toml

```diff
@@ -235,38 +235,38 @@
 "privateKeyPathDesc" = "Fill in an absolute path starting with '/'"
 "panelUrlPath" = "Panel URL Root Path"
 "panelUrlPathDesc" = "Must start with '/' and end with '/'"
-"pageSize" = "Pagination size"
+"pageSize" = "Pagination Size"
 "pageSizeDesc" = "Define page size for inbounds table. Set 0 to disable"
 "remarkModel" = "Remark Model and Seperation charachter"
-"sampleRemark" = "Sample remark"
+"sampleRemark" = "Sample Remark"
 "oldUsername" = "Current Username"
 "currentPassword" = "Current Password"
 "newUsername" = "New Username"
 "newPassword" = "New Password"
-"telegramBotEnable" = "Enable Telegram bot"
+"telegramBotEnable" = "Enable Telegram Bot"
 "telegramBotEnableDesc" = "Your telegram bot will interact with the panel"
 "telegramToken" = "Telegram Token"
-"telegramTokenDesc" = "The Token you have got from @BotFather"
-"telegramChatId" = "Telegram Admin ChatIDs"
-"telegramChatIdDesc" = "Multiple Chat IDs separated by comma. use @userinfobot or use '/id' command in bot to get your Chat IDs."
-"telegramNotifyTime" = "Telegram bot notification time"
-"telegramNotifyTimeDesc" = "Use Crontab timing format."
+"telegramTokenDesc" = "The token you have got from @BotFather"
+"telegramChatId" = "Telegram Admin chat IDs"
+"telegramChatIdDesc" = "Multiple chat IDs separated by comma. use @userinfobot or use '/id' command in bot to get your Chat IDs"
+"telegramNotifyTime" = "Telegram Bot Notification Time"
+"telegramNotifyTimeDesc" = "Use crontab timing format"
 "tgNotifyBackup" = "Database Backup"
 "tgNotifyBackupDesc" = "Send database backup file with report notification"
 "tgNotifyLogin" = "Login Notification"
-"tgNotifyLoginDesc" = "Displays the username, IP address, and time when someone tries to log into your panel."
-"sessionMaxAge" = "Session maximum age"
+"tgNotifyLoginDesc" = "Displays the username, IP address, and time when someone tries to log into your panel"
+"sessionMaxAge" = "Session Max Duration"
```

Comment:
`max age` it a term which should not be translated.
Also a better translation would be `Session duration`